### PR TITLE
Add docs for polkadot rest apis

### PR DESCRIPTION
**Pull request by pacmessica**

### CODE REVIEW

File: network-documentation/polkadot/rpc-and-rest-api/polkadot-rest-api.md [hunk 1]
```diff
@@ -0,0 +1,211 @@
...
+| **Parameter** | Type | Description |
+| :--- | :--- | :--- |
+| **stash_account** | string | account (required) |
+| **height** | int | block height (optional- if unspecified, returns latest) |
+
+
+**Example Request**
+
+```javascript
+polkadot--mainnet.datahub.figment.io/account/138QdRbUTB9eNY94Q4Mj5r39FkgMiyHCAy8UFMNA5gvtrfSB
```
Comment (pacmessica) [inline network-documentation/polkadot/rpc-and-rest-api/polkadot-rest-api.md:30]: guessed at these, not sure what the datahub is for the indexer

File: network-documentation/polkadot/rpc-and-rest-api/README.md [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+description: Learn what Polkadot APIs are available via DataHub and how to use them
+---
+
+# 🎮 RPC & REST API
+
+## Time to play
+
+Polkadot exposes a single RPC to enable the development of applications on the Polkadot network.
+
+Learn about the APIs available via Polkadot [**DataHub**](https://datahub.figment.io/sign_up?service=polkadot) ****below.
```
Comment (pacmessica) [inline network-documentation/polkadot/rpc-and-rest-api/README.md:11]: I don't think this exists yet

File: network-documentation/polkadot/rpc-and-rest-api/polkadot-rest-api.md (hunk 1 above)
Comment (pyr0gan) [reply network-documentation/polkadot/rpc-and-rest-api/polkadot-rest-api.md:30]: @guillaumegaluz  👍🏻

**Review by ajcronk — APPROVED** [review]
I will update the DH URLs for the services

**Pull request merged**